lib/Common/Io.hpp

```diff
@@ -144,22 +152,25 @@ class DOutPin : public IoPin
 
     /**
      * Read from digital output pin.
-     * 
+     *
      * @return Ditial output pin value.
      */
     int read() const
     {
-        return digitalRead(pinNo);
+        return (NC != m_pinNo) ? digitalRead(pinNo) : -1;

```

Comment:
Please follow Arduino: https://www.arduino.cc/reference/en/language/functions/digital-io/digitalread/
Recommended instead of -1 is LOW.

Reply:
fixed as suggested.